app/controllers/courses_controller.rb

```diff
@@ -0,0 +1,50 @@
+# Manages actions relating to editing and modifying
+# courses.
+class CoursesController < ApplicationController
```

Comment:
We only actually need the index and show methods here and we can delete the rest.